tests/ssl-opt.sh

```diff
@@ -1563,6 +1563,20 @@ run_test    "TLS 1.3, TLS_AES_128_CCM_8_SHA256, ext PSK, early data" \
 	    -s "<= parse early data"                        \
 	    -s "<= parse end_of_early_data"
 
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
+requires_config_enabled MBEDTLS_DEBUG_C
+requires_config_disabled MBEDTLS_RSA_C
+requires_config_enabled MBEDTLS_ZERO_RTT
+run_test    "TLS 1.3, TLS_AES_256_GCM_SHA384, ECDHE-ECDSA, SRV disables early data, client enables early data" \
+            "$P_SRV nbio=2 debug_level=5 force_version=tls1_3" \
+            "$P_CLI nbio=2 debug_level=5 force_version=tls1_3 force_ciphersuite=TLS_AES_256_GCM_SHA384 early_data=1" \
+            0 \
+      -s "Protocol is TLSv1.3"                                        \
+      -c "<= skip write early_data extension"                         \
```

Comment:
I am confused, shouldn't we see the client sending the early data extension?

Reply:
Updated the description. Client tries to send early data without PSK, it should not send any early data in such case, and it should fall back to 1-RTT.